Dockerfile

```diff
@@ -24,7 +24,8 @@ RUN apt-get update && apt-get install -y \
 	libgrpc++-dev \
 	libxi-dev \
 	libbz2-dev \
-	qtcreator
+	qtcreator \
```

Comment:
Unless you have found this to be essential, you can remove this line as was done on master as well.

Reply:
* [ ] Done